src/com/itschool/Session4/Course/Session4Class.java

```diff
@@ -0,0 +1,23 @@
+package com.itschool.Session4.Course;
+
+public class Session4Class {
+    public static void main(String[] args) {
+        System.out.println("Session 4 class!");
+
+        String firstName = "Alex";
+        String lastName = "Cristescu";
+
+        System.out.println(firstName + lastName);
+
+        firstName += lastName;
+        System.out.println(firstName);
+
+        boolean trueVar = true;
+
+        System.out.println("trueVar is " + !trueVar);
```

Comment:
this will be false here. I think in the course I wrote an extra line which prints a message something like "negation ..." and print the reversed value of this boolean.